apis/kueue/v1beta1/workload_types.go

```diff
@@ -587,6 +587,9 @@ const (
 	// when the Pods were ready since the workload admission, but some pod has failed,
 	// and workload waits for recovering.
 	WorkloadWaitForPodsRecovery = "WaitForPodsRecovery"
+
+	// WorkloadNotAdmitted is a reason that indicates the Workload has not been admitted
```

Comment:
Is this reason used for workload which get evicted and are waiting to be re-admitted?
If no, then maybe say "the Workload has never been admitted yet" to make it more explicit, wdyt?

Reply:
It is used for both cases - first admission and readmission

Reply:
So, then maybe say "... the Workload is not admitted". I'm not an English expert, but " has not been admitted" sounds ambiguous to me, and I actually assumed you mean "has never been admitted yet".

Reply:
Oh yes, done